mungegithub/mungers/submit-queue.go

```diff
@@ -67,6 +67,13 @@ var (
 	_ = fmt.Print
 	// This MUST cause a RETEST of everything in the sq.RequiredRetestContexts
 	retestBody = fmt.Sprintf("@%s test this [submit-queue is verifying that this PR is safe to merge]", jenkinsBotName)
+
+	requiredContexts = []string{
```

Comment:
we need to make these all flags somehow eventually if we want to run this thing against other repos.